src/test/java/org/apache/commons/lang3/ArrayUtilsTest.java

```diff
@@ -2087,7 +2087,730 @@ public void testReverseObjectRange() {
         assertEquals(null, array);
     }
     
+    //-----------------------------------------------------------------------
```

Comment:
I did the tests this way to be in sync with all the other tests in ArrayUtilsTest.